File: 998_Four_Integers/main.py
def main(i: str):
  l1 = sorted([int(x) for x in i.split(' ')])
  l2 = [x + 1 for x in range(min(l1) - 1, max(l1))]
  print('Yes' if l1 == l2 else 'No')

if __name__ == '__main__':
  i: str = input()
#   i = "30 31 32 33 34"
  main(i)

File: 998_Four_Integers/main.cs
using System;

String input = Console.ReadLine();
// String input = "1 2 3 4";
IEnumerable<int> l1 = input.Split(" ")
                      .Select(s => Convert.ToInt32(s))
                      .OrderBy(i => i);
IEnumerable<int> l2 = Enumerable.Range(l1.Min(), l1.Max());

Console.WriteLine(l1.SequenceEqual(l2)? "Yes" : "No");

